tests/functional/conftest.py

```diff
@@ -39,6 +39,13 @@ def internet_explorer(selenium):
     return selenium.capabilities.get('browserName').lower() == 'internet explorer'
 
 
+@pytest.fixture
+def ie9(selenium):
```

Comment:
@alexgibson this skip is not triggered by ie9. Can you please have a look or recommend someone to help me with it?

Reply:
You shouldn’t need to add any new code in order to remove a test from being run in IE9. Removing the `sanity` marker from the test should do it.